### ISSUE TITLE
commit not work

### ISSUE THREAD

**Issue by Zaazik** [birdayz/kaf#91]
```$ kaf group commit consumer_group -t=some.topic-p=0 -o=137638```
```Set offset to 137638.%```
```$ kaf group describe consumer_group```
```Group ID:        consumer_group
State:           Empty
Protocol:
Protocol Type:   consumer
Offsets:
  some.topic:
    Partition  Group Offset  High Watermark  Lag    Metadata
    ---------  ------------  --------------  ---    --------
    0          104959        137639          32680
    1          104835        134117          29282```

**Comment by birdayz** [comment]
thanks for reporting. i experience the same..the join is not performed correctly. apparently the group is stuck in Preparing/CompletingRebalance because i don't send some specific packet.
i got a quick fix for it; i'll have it ready this evening i suppose.

**Comment by birdayz** [comment]
released v0.1.28 with a fix. enjoy

**Comment by Zaazik** [comment]
🤗
